libgadget/density.c

```diff
@@ -55,6 +55,7 @@ typedef struct {
 
 struct DensityPriv {
     MyFloat *Left, *Right;
+    MyFloat *Rot;
```

Comment:
MyFloat (*Rot)[3], and sizeof can be  sizeof(priv->Rot[0])
Then you can get get rid of those 3* formality in array indexing (easier to align lines without them.)

Reply:
To case malloc result use (MyFloat (*) [3])

Reply:
Done, I think. Like 9181ae3 ?